### DISCUSSION TITLE
Grape harvesting large vine yards

### DISCUSSION THREAD

**Discussion by Kiraxon** [Courseplay/Courseplay_FS22#1869] [Q&A] [answered]
I was wondering if there is any new improvements to vine yard harvesting or methods to automate vineyard harvesting. For large vine yards I am inquiring about.

I noticed CP does have an option to harvest grapes and olives automatically, but when the harvest fills up, the harvester just stops. I tried harvester self unload to see if maybe it might unload but it doesn't seem to do that either. 

I can imagine there might be some challenges as the grape fines probably act as a colission and can make it difficult exiting the vine line and of course returning to the vine line. 

Its a long shot on my questions but I thought I would post and see if anyone had gotten something to work with CP or AD. Especially not that there is the new Grape harvester out that side unloads as opposed to the New Holdand harvester that reverse unloads.

Regards,

**Comment by Tensuko** [comment, marked as answer]
We have plans to further improove this. Right now our focus is on something else so this got to wait a bit more.

**Reply by Kiraxon** [reply]
Tensuko, thank you for the reply. I am glad it's something you guys are considering for the future.

Cheers,

**Comment by sirupflex** [comment]
Wonderful to hear that the team is considering the automation of vineyard harvester unloading.  It is my biggest painpoint left, all other key elements have been addressed by CP already so amazingly. Really love it, it is fantastic what you all have accomplished, thank you!! Once there are news on this topic please let us know.

**Comment by rokatopo** [comment]
Was this ever worked on? Curious as I'm planning to go all in on olives on a map server. Thanks :)

**Reply by Tensuko** [reply]
Not at the moment.
